build.gradle

```diff
@@ -94,6 +94,17 @@ subprojects {
             implementation 'net.sourceforge.streamsupport:streamsupport-flow:' + rootProject.ext.streamSupportVersion
             implementation 'net.sourceforge.streamsupport:streamsupport-literal:' + rootProject.ext.streamSupportVersion
 
+            /*add music dependencies
```

Comment:
These dependencies are only relevent for swing. So this belongs in go.graphics.swing